lib/web_of_science/harvester.rb

```diff
@@ -73,11 +72,15 @@ def process_uids(author, uids)
       raise(ArgumentError, 'uids must be Enumerable') unless uids.is_a? Enumerable
       uids.reject! { |uid| publication_identifier?('WosItemId', wos_item(uid)) }
       return [] if uids.empty?
-      process_records author, wos_queries.retrieve_by_id(uids)
+      process_records author, queries.retrieve_by_id(uids)
     end
 
     private
 
+      delegate :logger, to: :WebOfScience
+
+      delegate :queries, to: :WebOfScience
```

Comment:
Could combine these 2 lines, if you like:  
```ruby
delegate :logger, :queries, to: :WebOfScience
```

Reply:
Pushed.